src/driver/xdma_sgdma.c

```diff
@@ -115,118 +115,6 @@ static int check_transfer_align(struct xdma_engine *engine,
 	return 0;
 }
 
-/*
- * Map a user memory range into a scatterlist
- * inspired by vhost_scsi_map_to_sgl()
- * Returns the number of scatterlist entries used or -errno on error.
- */
-
-static void xdma_channel_unmap_user_buf(struct xdma_io_cb *cb, bool write)
```

Comment:
Do earlier commits not fail when compiling with unused static functions?

I'd probably squash this with the earlier commit. Would have made reviewing a bit easier too as I could have compared this code with the new code.

Reply:
A warning only, but I'll squash it for the future iteration